android/source/VideoLocker/build.gradle

```diff
@@ -232,7 +232,7 @@ android {
 
             versionCode 40
             versionName '1.0.03'
-            minSdkVersion 16
+            minSdkVersion 14
             targetSdkVersion 21
```

Comment:
@rohan-dhamal-clarice - These are two different flavours of builds(default and prodRelease). Hence I suggest that instead of taking the common variable, we let it be different for now.